samples/openthread/cli/README.rst

```diff
@@ -64,20 +64,21 @@ See `Testing diagnostic module`_ section for an example.
 .. note::
     If you disable the :kconfig:option:`CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER` feature set, you can enable the diagnostic module with the :kconfig:option:`CONFIG_OPENTHREAD_DIAG` Kconfig option.
 
-For the ``nrf52840dongle_nrf52840`` build target, diagnostic GPIO commands can also be used.
-They are enabled in the ``openthread_config`` node in the :file:`boards/nrf52840dongle_nrf52840.overlay` file, with the node configured for the **P0.19** pin, which is connected to the **RESET** pin on the nRF52840 Dongle.
-The pin is set in output mode with a low state so that the **RESET** pin is pulled to **GND**, which results in the device rebooting without skipping the bootloader.
+.. _ot_cli_sample_bootloader:
+
+Rebooting to bootloader
+=======================
+
+For the ``nrf52840dongle_nrf52840`` build target, the device can reboot to bootloader by triggering a GPIO pin.
+To enable this behavior, enable the :kconfig:option:`CONFIG_OPENTHREAD_PLATFORM_BOOTLOADER_MODE_GPIO` Kconfig option and configure the Devicetree overlay in the :file:`boards/nrf52840dongle_nrf52840.overlay` file.
+For this sample the ``bootloader-gpios`` property in the ``openthread_config`` node is configured for the **P0.19** pin, which is connected to the **RESET** pin on the nRF52840 Dongle.
```

Comment:
```suggestion
For this sample, the ``bootloader-gpios`` property in the ``openthread_config`` node is pre-configured for the **P0.19** pin, which is connected to the **RESET** pin on the nRF52840 Dongle.
```
I think changing it to "pre-configured" might be good, so it doesn't seem at odds with telling the user to configure the Devicetree overlay.

Reply:
Makes sense, fixed.